### PR TITLE
update topology documentation

### PR DESCRIPTION
**Pull request by adityathebe**
resolves: https://github.com/flanksource/docs/issues/180

### CODE REVIEW

File: mission-control/docs/topology/concepts/lookup.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                    labelSelector: 'app.kubernetes.io/name=ingress-{{.component.name}}&app.kubernetes.io/component=controller'
+      // highlight-end
+```
+
+| Field           | Description                                     | Scheme                                                       | Required |
+| --------------- | ----------------------------------------------- | ------------------------------------------------------------ | -------- |
+| `components`    | Create sub-components for each component        | [`[]Component`](../references/components.md)                 |          |
+| `properties`    | Create or lookup properties for each component  | [`[]Property`](./properties.md)                              |          |
+| `configs`       | Link configuration items for each component     | [`[]ConfigSelector`](./catalog.md#config-selector)           |          |
+| `checks`        | Create or link health checks for each component | [`[]CheckSelector`](./health-checks.md#check)                |          |
```
Comment (moshloop) [inline mission-control/docs/topology/concepts/lookup.md:89]: Check column descriptions

File: mission-control/docs/topology/concepts/lookup.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                  resource:
+                    labelSelector: 'app.kubernetes.io/name=ingress-{{.component.name}}&app.kubernetes.io/component=controller'
+      // highlight-end
+```
+
+| Field           | Description                                     | Scheme                                                       | Required |
+| --------------- | ----------------------------------------------- | ------------------------------------------------------------ | -------- |
+| `components`    | Create sub-components for each component        | [`[]Component`](../references/components.md)                 |          |
+| `properties`    | Create or lookup properties for each component  | [`[]Property`](./properties.md)                              |          |
+| `configs`       | Link configuration items for each component     | [`[]ConfigSelector`](./catalog.md#config-selector)           |          |
```
Comment (moshloop) [inline mission-control/docs/topology/concepts/lookup.md:88]: Isn't this a resource selector ?

File: mission-control/docs/topology/references/exec.md [hunk 3]
```diff
@@ -1,33 +1,48 @@
...
-  interval: 30
-  exec:
-  - description: "exec dummy check"
-    script: |
-      echo "hello"
-    name: exec-pass-check
-    test:
-      expr: 'results.Stdout == "hello"'
+  type: Pods
+  icon: kubernetes
```
Comment (moshloop) [inline mission-control/docs/topology/references/exec.md:20]: This example isn't very practical , using an exec plugin to create a Server component by pulling data from uname, ifconfig, etc would be more practical

File: mission-control/docs/topology/concepts/lookup.md (hunk 2 above)
Comment (adityathebe) [reply mission-control/docs/topology/concepts/lookup.md:88]: It's not. It's a `ConfigQuery`

**Pull request merged**